src/DssSpell.t.base.sol

```diff
@@ -7,6 +7,7 @@ import "ds-test/test.sol";
 import "dss-interfaces/Interfaces.sol";
 import "./test/rates.sol";
 import "./test/addresses_mainnet.sol";
+import "./test/addresses_deployers.sol";
```

Comment:
consider adding import for `addresses_wallets.sol` as well.

Reply:
even if we are not using it currently?

Reply:
yeah, I would import it for convenience following a similar reasoning as per `addresses_mainnet` for specific tests (eg. `testNewChainlogValues`) to be used and updated when needed.